content/blog/2022/11/2022-11-14-hacktoberfest-recap.adoc

```diff
@@ -0,0 +1,159 @@
+---
+layout: post
+title: "A Wonderful Hacktoberfest in Jenkins"
+tags:
+- hacktoberfest
+- event
+- community
+- newcomer
+- outreach-programs
+author: kmartens27
+description: >
+  Hacktoberfest 2022 was a great success for the Jenkins project.
+  This post highlights Jenkins contributions in artwork, translation, documentation, security, and new features.
```

Comment:
I wasn't sure if we should use localization instead of translation?
```suggestion
  This post highlights Jenkins contributions in artwork, localization, documentation, security, and new features.
```

Reply:
Both are valid contributions during Hacktoberfest.  I tend to prefer "translation" because it more precisely represents the action performed by the contributor.

Translation pull requests included https://github.com/jenkinsci/jenkins/pull/7230 (Russian), https://github.com/jenkinsci/jenkins/pull/7272 (Russian),  https://github.com/jenkinsci/jenkins/pull/7255 (Brazilian Portugese).

Based on the [Wikipedia definition of localization](https://en.wikipedia.org/wiki/Internationalization_and_localization) I believe that either word could be used here.  The pull requests were translating from English into another language.  They were also adapting the software for use in that other language.

I don't recall seeing any internationalization pull requests, though I didn't see every pull  request.  If there are examples of internationalization pull requests,, then I think that we could also add "internationalization" to the list of pull request types.

For precision, I'm acting as though "translation" is the conversion of user visible text in English to text in another language.  In Jenkins, "translation" is largely a synonym of "localization" because we don't have much support for other localization techniques.

Reply:
agreeing with @MarkEWaite 's point of view